Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -61,130 +68,76 @@ public partial class Autocomplete<TItem, TValue> : BaseAfterRenderComponent, ICl
         /// </summary>
         bool canShowDropDown;
 
-        #endregion
+        string currentSearch;
```

Comment:
>Reworked SetParametersAsync to use 3 variables pattern. Is it better?…
… Who knows..

So if you don't know if it's better, why risk changing it? What's better that makes it a good choice over the change to introduce new bugs.

We all know it's hard to be constantly changing stuff without introducing subtle bugs, we aren't machines haha

Reply:
That was a rethorical question :) I think it's better because it's less code and it's more clear, sorry for confusion :)